ee/tabby-webserver/src/service/background_job/web_crawler.rs

```diff
@@ -14,7 +14,7 @@ use super::helper::Job;
 
 const CRAWLER_TIMEOUT_SECS: u64 = 7200;
 
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize, Clone)]
```

Comment:
is this still necessary?

Reply:
yes, `event` is used to create a readable name for notifications, and run jobs, borrowing would cause many lifetime changes for different kinds of jobs, cloning is the simplest solution currently.

when event is clonable, this also need to be clonable